tests/integration/components/project-list-test.js

```diff
@@ -21,8 +36,27 @@ test('it renders an item for each project in the list', function(assert) {
     mockProjects.push({ id: i, title: `Project ${i}`, slug: `project_${i}` });
   }
 
-  this.set('projects', mockProjects);
+  set(this, 'projects', mockProjects);
   this.render(hbs`{{project-list projects=projects}}`);
 
   assert.equal(this.$('.project-item').length, 3, 'The correct number of project-items is rendered');
 });
+
+test('it sorts the list by id', function(assert) {
+  assert.expect(3);
+
+  let mockProjects = [];
+  let mockProjectIds = [1, 3, 2];
+
+  mockProjectIds.forEach(function(id) {
+    mockProjects.push({ id, title: `Project ${id}`, slug: `project_${id}` });
+  });
+
+  set(this, 'projects', mockProjects);
+  page.render(hbs`{{project-list projects=projects}}`);
+
+  for (let i = 0; i < 3; i++) {
+    let id = i + 1;
```

Comment:
I think you could do a little small improvement here, just remove the id line and change the for to `for (let id = 1; id <= 3; id++) {`. 

Nice work! 👍